admin-core/src/main/resources/application.conf

```diff
@@ -2,18 +2,16 @@ ktor {
     deployment {
         port = 8090
```

Comment:
I don't see how we can disable this port via configuration.
Lets think of something